jerry-core/jerry-port.h

```diff
@@ -35,6 +35,40 @@ int jerry_port_logmsg (FILE *stream, const char *format, ...);
 int jerry_port_errormsg (const char *format, ...);
 int jerry_port_putchar (int c);
 
+/*
+ * Termination Port API
+ *
+ * Note: It is questionable whether a library should be able to terminate an
```

Comment:
Minor style issue. The common style for note comments are that using newline and indent to colon.

```C++
/*
 * Note:
 *      It is questionable whether...
```